content/en/docs/developer_guide/flows/flow_triggers/oas_flow_trigger.md

```diff
@@ -54,7 +54,33 @@ If all your designed paths begin with a common prefix, i.e. `/service`, then by
 The OpenAPI specification is bound to `/apidoc/swagger.json`, `/apidoc/swagger.yaml` (both for legacy purposes), as well as `/apidoc/openapi.json` and `/apidoc/openapi.yaml`. On startup, users will see only one path in the log. For OpenAPI 2.0, it is `/apidoc/swagger.json`, otherwise, it is `/apidoc/openapi.json`.
 The prefix (`/apidoc`) is configured by changing [`apidoc.prefix` in configuration](/docs/developer_guide/project/configuration/project_configuration#apidoc).
 
-## Currently unsupported features
+<!-- ## Request handling
+Describe what we do on an inbound request before hitting the flow. Write about what validation, massaging, coercing, response codes to expect and when. -->
+
+## Response handling
+
+{{% variables/apibuilder_prod_name %}} and the OpenAPI flow-trigger do additional processing after a flow completes, before sending the response. This section describes the areas that are processed.
+
+## Response body
+You can send a `string` or `Buffer` as the response body in a HTTP request.
+
+When a different data type is set as the response body, we may encode it as JSON if your OpenAPI spec documents a _single_ `application/json` type response for that status code (or default if the code in use is not documented). Otherwise, the encoding is ambiguous so results in a 500 Internal Server Error. In this case, you should ensure you encode the body within the flow and set an appropriate `content-type` header for the encoding used.
```

Comment:
```suggestion
When the response body is a type _other_ than `string` or `Buffer`, and your OpenAPI spec documents a _single_ `application/json` type response for that status code (or `default` if the code in use is not documented), then the response body will be JSON encoded. Otherwise, the response content encoding is ambiguous, and results in a `500 Internal Server Error`. In this case, you should ensure you encode the body within the flow and set an appropriate `content-type` header for the encoding used.
```